tests/test_environments/values/ci-eks-config-1.yml

```diff
@@ -2,6 +2,9 @@ elasticsearch:
   volumeClaimTemplate:
     storageClassName: gp2
 
+  nodeSelector:
+    "elasticsearch-node": "true"
```

Comment:
Since a custom configuration is defined on one of the EKS nodes, pinning the Elasticsearch deployment to a specific node solves the issue with the failing Elasticsearch deployment.